contracts/FlowTransactionScheduler.cdc

```diff
@@ -673,7 +673,10 @@ access(all) contract FlowTransactionScheduler {
             // Calculate the FLOW required to pay for storage of the transaction data
             let storageFee = FlowStorageFees.storageCapacityToFlow(dataSizeMB)
             
-            return scaledExecutionFee + storageFee
+            // Add inclusion Flow fee for scheduled transactions
```

Comment:
Wouldn't this be better to just include in the `FlowFees.computeFees` call in `calculateFees()`. We can just use a higher inclusion effort there

Reply:
I guess I just don't understand the purpose of this. Because the minimum execution effort is 100, there is always a minimum fee, so this feel unnecessary

Reply:
Check comment from Dete asking for this and his explanation. https://gololicic.slack.com/archives/C08LL92U3BK/p1760473092311309

Reply:
Yes, I saw it. We can include it in the next upgrade